### ISSUE TITLE
train.record

### ISSUE THREAD

**Issue by ebraheemsaleh** [yinguobing/cnn-facial-landmark#104]
Which dataset train.record should point to?
Dataset of box annotated faces ?

**Comment by yinguobing** [comment]
Actually face images and marks.

**Comment by CardLin** [comment]
I am using face-mesh-generator's features/tfrecord-marks-68 branch to generate dataset:
300vw, 300w, AFLW2000_3D, afw, helen, lfpw as train.record
ibug as val.record
Training 1epochs with lr=0.01, 10epochs with lr=0.001, 100epochs with lr=0.0001
After training complete, I also transfer model to tensorflow frozen model (.pb only)
What I found is that the prediction of mouth is very good. But the prediction of eye is very worse.
The prediction result of the eye always open, even I close my eye.
Any suggestion?

**Comment by yinguobing** [comment]
It is time to move on to a more advanced project -> https://github.com/yinguobing/facial-landmark-detection-hrnet

If you would like to stay with this one, I recommand starting with weighted loss function.

Please open a new issue next time. I think this issue had gone stale and better to be closed.
